Assets/HoloToolkit/UX/Scripts/Lines/LineUtility.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MRTK.UX
+{
+    #region structs
+    [Serializable]
+    public struct SplinePoint
+    {
+        public Vector3 Point;
+        public Quaternion Rotation;
+    }
+    #endregion
+
+    public static class LineUtils
+    {
+        #region enums
+
+        public static readonly Vector3 DefaultUpVector = Vector3.up;
+
+        public enum InterpolationEnum
```

Comment:
Each of the enums in this class could be in their own files as well, and probably don't need to be in this `LineUtils` class

Reply:
Moved all enums out of the LineUtils class.

We should still keep them in a single file, either LineUtility.cs, or possible in LineBase.cs alongside the LineBase class. Or maybe we could compromise and put them in a single enum-only class called LineBaseEnums.cs.

Here's my reasoning - clumping enums together with the class that interprets them is informative. It's a form of documentation that says: these things all related - taken together, they form a complete picture of this object's possible behavior.

Breaking them up into separate files destroys that information while adding complexity. Devs have to work backwards from variables one-by-one, forensically reconstructing relationships that should be clear at a moment's glance.